File: 1922.cpp
#include <stdio.h>
#include <vector>

using namespace std;

struct Vertex {
	int t;
	int d; 
};

int main() {
	
	int N, M;
	int a,b,c;
	vector<Vertex> list[1001];
	
	freopen("inputs/1922.txt","r",stdin);
	
	scanf("%d",&N);
	scanf("%d",&M);
	
	
	for(int i = 0; i < N ; i++) {
		scanf("%d %d %d",&a, &b, &c);
		list[a].push_back(Vertex(b,c))
	}
	
	
	printf(N);
	
	
	
}

File: 1922.py

import sys
sys.stdin = open("inputs/1922.txt") 


class V :

    def __init__(self, a, b) :
        self.a = a
        self.b = b

    def __str__(self) :
        return "%s %s" % (self.a , self.b)

N = int(input())
M = int(input()) 

for i in range(0,N) :
    a,b,c = map(int,input().split())
    list[a][b] = c
    list[b][a] = c

